shared/common-adapters/emoji.desktop.js

```diff
@@ -10,9 +10,22 @@ import type {Props} from './emoji'
 
 const backgroundImageFn = (set: string, sheetSize: string) => emojiSet
 
+const skinToneRegex = /:([^:]*):(:skin-tone-\d:)?/i
```

Comment:
This regexp is slightly different from https://github.com/missive/emoji-mart/blob/9cd6494c394add8283cabc03dd7823e3a39041e7/src/utils/index.js#L7 (ours is case-insensitive, doesn't match entire string) -- perhaps we should match it and have a comment saying where it comes from?

Reply:
Also, the name is slightly confusing, because it's a general emoji regexp, right?

Reply:
I didn't base this off of that link, I just wrote this regex to extract what's between the first pair of colons. But sure, I can change the name